src/loader.js

```diff
@@ -0,0 +1,5 @@
+function load(lib) {
+    return import(`./equations/${lib}`)
```

Comment:
Getting this relative to work was a real PITA.  This loader is on the root level and makes life easier.  Not sure what is special about `import()`.

Maybe its worth using webpack just to get rid of the relative imports? I like that a lot. Multiple builds could be nice too.